diff --git a/include/v8-local-handle.h b/include/v8-local-handle.h
index 03f7d5212e4d..98feeeba76d4 100644
--- a/include/v8-local-handle.h
+++ b/include/v8-local-handle.h
@@ -377,8 +377,7 @@ class V8_TRIVIAL_ABI Local : public LocalBase<T>,
   explicit Local(const Local<T>& other, no_checking_tag do_not_check)
       : LocalBase<T>(other), StackAllocated(do_not_check) {}
 
-  V8_INLINE explicit Local<T>(const LocalBase<T>& other)
-      : LocalBase<T>(other) {}
+  V8_INLINE explicit Local(const LocalBase<T>& other) : LocalBase<T>(other) {}
 
   V8_INLINE static Local<T> FromSlot(internal::Address* slot) {
     return Local<T>(LocalBase<T>::FromSlot(slot));
